public/500.html

```diff
@@ -1,66 +1,53 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>We're sorry, but something went wrong (500)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  body {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
+  <head>
+    <title>We're sorry, but something went wrong (500)</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <style>
+      body {
+          background-color: #EFEFEF;
+          color: #2E2F30;
+          font-family: arial, sans-serif;
+          margin: 0;
+          width: 100vw;
+          height: 100vh;
+      }
+      picture, img {
+          position: fixed;
+          top: 0;
+          left: 0;
+	  z-index: 0;
 
-  div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
+          /* Preserve aspet ratio */
+          min-width: 100%;
+          min-height: 100%;
+      }
 
-  div.dialog > div {
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #BBB;
-    border-top: #B00100 solid 4px;
-    border-top-left-radius: 9px;
-    border-top-right-radius: 9px;
-    background-color: white;
-    padding: 7px 12% 0;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
+      #content {
+          z-index: 1;
+          position: relative;
+          padding: 5%;
+      }
+    </style>
+  </head>
 
-  h1 {
-    font-size: 100%;
-    color: #730E15;
-    line-height: 1.5em;
-  }
-
-  div.dialog > p {
-    margin: 0 0 1em;
-    padding: 1em;
-    background-color: #F7F7F7;
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #999;
-    border-bottom-left-radius: 4px;
-    border-bottom-right-radius: 4px;
-    border-top-color: #DADADA;
-    color: #666;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
-  </style>
-</head>
-
-<body>
-  <!-- This file lives in public/500.html -->
-  <div class="dialog">
-    <div>
-      <h1>We're sorry, but something went wrong.</h1>
+  <body>
+    <picture>
+      <source media="(min-width: 36em)"
+              srcset="500_assets/full_size.jpg  2667w,
+                      500_assets/large.jpg  1024w,
+                      500_assets/medium.jpg 640w,
+                      500_assets/small.jpg  320w"
+              sizes="100vw" />
+      <source srcset="500_assets/cropped-large.jpg 2x,
+                      500_assets/cropped-medium.jpg 1.5x,
+                      500_assets/cropped-small.jpg 1x" />
+      <img src="500_assets/small.jpg" />
```

Comment:
I think I'd prefer if there was a directory called `errors` or `error-pages` with `500` and `404` directories inside of it.

Reply:
sounds good.

just a note, 404 will never be seen (we have that global re-direct to the homepage)

Reply:
Good point!

Though, I think it's maybe better to land on a 404 (and report it in the logs) than just opaquely redirect to root. I think I initially made that root redirect when things were still messy and in flux at the start. Getting rid of it might help identify where things ARE actually missing or broken.

What do you think about removing it and adding a static 404?

Reply:
@veganstraightedge agreed. I will drop the redirect as part of the 404 PR